src/background.js

```diff
@@ -1,2 +1,67 @@
-browser.blurts.start();
-browser.blurts.onTelemetryEvent.addListener((id) => console.log(id));
+let gTelemetryListener = async function(event) {
+  console.log(event);
+  browser.study.sendTelemetry({event});
+}
+
+async function init() {
+  browser.study.onEndStudy.addListener((ending) => {
+
+  });
+  browser.study.onReady.addListener((studyInfo) => {
+    browser.blurts.start(studyInfo.variation.name);
+    browser.blurts.onTelemetryEvent.addListener(gTelemetryListener);
+  });
+  await browser.study.setup({
+    allowEnroll: true,
+    activeExperimentName: browser.runtime.id,
+    studyType: "shield",
+    telemetry: {
+      send: true,
+      removeTestingFlag: false,
+    },
+    weightedVariations: [
+      {
+        name: "1",
+        weight: 1,
+      },
+      {
+        name: "2",
+        weight: 1,
+      },
+      {
+        name: "3",
+        weight: 1,
+      },
+      {
+        name: "4",
+        weight: 1,
+      },
+      {
+        name: "5",
+        weight: 1,
+      },
+    ],
+    endings: {
+      "user-disable": {
+        baseUrls: [
+          "https://qsurvey.mozilla.com/s3/Shield-Study-Example-Survey/?reason=user-disable",
```

Comment:
I assume this URL is just a placeholder (since the name says "example survey"). Not sure if it's worth turning into some `const` variable, so we won't need to update the URL in 3+ places when we get a final URL.